examples/FloatImage.ipynb

```diff
@@ -25,7 +25,8 @@
     "\n",
     "\n",
     "url = (\n",
-    "    \"https://github.com/ocefpaf/secoora_assets_map/blob/a250729bbcf2ddd12f46912d36c33f7539131bec/secoora_icons/rose.png\"\n",
+    "    \"https://raw.githubusercontent.com/ocefpaf/secoora_assets_map/\"\n",
```

Comment:
Thanks for the permalink. Note that we need the raw URL here to display the image.